index.html

```diff
@@ -26,36 +26,191 @@ <h4>Rabaya Rabu</h4>
                     don’t hestiate to contact me.
                 </p>
                 <!-- social media section -->
-                    <ul class="image-icons">
-                        <li><a href="https://github.com/rabayarabu"><img src="Images/GitHub.png"
-                                    alt="GitHub"></a></li>
-                        <li><a href="https://www.linkedin.com/in/rabaya-rabu-142721169/"><img src="Images/Linkedin.png"
-                                    alt="Linkedin"></a></li>
-                        <li><a href=""><img src="Images/Angelist.png"
-                                    alt="Angelist"></a></li>
-                        <li><a href="https://twitter.com/rabaya_rabu"><img src="Images/Twitter.png"
-                                    alt="Twitter"></a></li>
-                        <li><a href=""><img src="Images/Medium.png"
-                                    alt="Medium"></a></li>
+                <ul class="image-icons">
+                    <li><a href="https://github.com/rabayarabu"><img
+                                src="Images/GitHub.png"
+                                alt="GitHub"></a></li>
+                    <li><a
+                            href="https://www.linkedin.com/in/rabaya-rabu-142721169/"><img
+                                src="Images/Linkedin.png"
+                                alt="Linkedin"></a></li>
+                    <li><a href=""><img src="Images/Angelist.png"
+                                alt="Angelist"></a></li>
+                    <li><a href="https://twitter.com/rabaya_rabu"><img
+                                src="Images/Twitter.png"
+                                alt="Twitter"></a></li>
+                    <li><a href=""><img src="Images/Medium.png"
+                                alt="Medium"></a></li>
+                </ul>
+            </div>
+        </section>
+        <!-- works section -->
+        <section class="works">
+            <article class="grid-container">
+                <h1>My Recent Works</h1>
+                <div class="devider1">
+                    <img src="Images/Vector.png" alt="vector">
+                </div>
+                <div class="image"></div>
+                <div class="card-body">
+                    <h5>Multi-Post Stories</h5>
+                    <p>A daily selection of privately
+                        personalized reads; no accounts or sign-ups required.
+                        has been the industry's standard dummy text ever since
+                        the 1500s,
+                        when an unknown printer took a standard dummy text.
+                    </p>
+                    <ul>
+                        <li class="css">CSS</li>
+                        <li class="html">HTML</li>
+                        <li class="bootstrap">Bootstrap</li>
+                        <li class="ruby">Ruby</li>
+                    </ul>
+                    <button type="button">See project</button>
```

Comment:
Kindly update **all** buttons on your page to have button interaction styling when hovered and pressed. You can use the `:hover` and `:active` pseudo selectors to achieve this. When doing so, please stick to the figma design guidelines for button styling👍:
<img width="447" alt="image" src="https://user-images.githubusercontent.com/37116322/227625333-8e41a397-0644-476c-917a-a378863d2b1b.png">